content/en/opentelemetry/reference/concepts.md

```diff
@@ -14,14 +16,14 @@ further_reading:
 
 This page describes essential terms and concepts for OpenTelemetry and Datadog. For additional definitions and descriptions, see the [OpenTelemetry Glossary][6].
 
-| Concept                      | Description                                                                                                                                                                                                                                                               |
-|------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Telemetry                    | The collection of metrics, logs, and traces that provide observations about the behaviors and performance of applications and infrastructure.                                                                                                                             |
-| [OpenTelemetry Collector][1] | A vendor-agnostic implementation for collecting and exporting telemetry data emitted by many processes. It can be configured to receive, process, and export telemetry to one or multiple destinations including storage backends and analysis tools.                      |
-| [Datadog Exporter][2]        | A component that lets you forward trace, metric, and logs data from OpenTelemetry SDKs to Datadog.                                                                                                                                                                     |
+| Concept                      | Description                                                                                                                                                                                                                                                      |
+|------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Telemetry                    | The collection of metrics, logs, and traces that provide observations about the behaviors and performance of applications and infrastructure.                                                                                                                    |
```

Comment:
Maybe worth introducing profiles here? We mention it as one of the proprietary features that customers can use, yet we don't include it as part of 'telemetry' here